src/ElasticSearchRepository.php

```diff
@@ -44,28 +46,29 @@ public function __construct(
         Client $client,
         Serializer $serializer,
         string $index,
-        string $class,
+        string $class_type,
         array $notAnalyzedFields = []
     ) {
         $this->client = $client;
         $this->serializer = $serializer;
         $this->index = $index;
-        $this->class = $class;
+        $this->class_type = $class_type;
         $this->notAnalyzedFields = $notAnalyzedFields;
     }
 
     /**
-     * {@inheritdoc}
+     * @throws AssertionFailedException
      */
     public function save(Identifiable $data): void
     {
-        Assertion::isInstanceOf($data, $this->class);
+        Assertion::true(
```

Comment:
Does the assertion library not support providing error messages for the `isInstanceOf` test?

Reply:
It does indeed. I was tricked by the PHP Inspections of the IDE recommending me this option to avoid some edge case with the string type of the class. Never mind, getting it back :)

Reply:
More info coming from the phpstan analyzer: 
 67     Parameter #2 $className of static method Assert\Assertion::isInstanceOf() expects class-string<object>, string given.